src/js/models/messages.js

```diff
@@ -65,7 +69,7 @@ export default class Messages{
 		try{
 			await db.collection('Messages').doc(chatId).collection('history').add({
 				content: message,
-				senderId: this.selfPos,
+				senderID: this.selfPos,
```

Comment:
If you're going to change `Id` to `ID`, you have to change it everywhere (like [here](https://github.com/Yuxuan-Lin/VioLegacy/pull/33/files#diff-8ff12f9af9c29ec28325c2225dfa9adfa12627d678c2db8a31f9c382ece8af29R21)). Otherwise, don't change it.

Reply:
We have already made sure that consistency.

Reply:
It still says `chatterId`

Reply:
Problem resolved in new version.